projects/packages/jetpack-mu-wpcom/src/features/wpcom-user-edit/wpcom-user-edit.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * Prevent the site owner from editing user's account-level fields.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+namespace Automattic\Jetpack\Jetpack_Mu_Wpcom;
+
+require_once __DIR__ . '/../../utils.php';
+
+/**
+ * Disable the account-level fields of the connected users to prevent the site owner from editing them.
+ */
+function wpcom_disable_account_level_fields_if_needed() {
+	if ( is_super_admin() ) {
```

Comment:
Hmm... on Atomic sites, it looks like site owner will have `is_super_admin()` == `true`?

Administrators on my test Atomic site is still able to modify connected users... could you double check?

Reply:
Hmmm...

Reply:
I changed to use `is_network_admin()` as the original message is to edit the user from the network admin 🫣

Reply:
Updated the test instructions for the Network Admin!